src/index.html

```diff
@@ -1,13 +1,420 @@
+              </div>
+            </div>
+          </div>
+
+          <ul class="menu__list">
+            <li class="menu__item">
+              <a href="#" class="menu__link">Home</a>
+            </li>
+
+            <li class="menu__item">
+              <a href="#about-us" class="menu__link">About Us</a>
+            </li>
+
+            <li class="menu__item">
+              <a href="#compare-bikes" class="menu__link">Compare bikes</a>
+            </li>
+
+            <li class="menu__item">
+              <a href="#details" class="menu__link">Details</a>
+            </li>
+
+            <li class="menu__item">
+              <a href="#contacts" class="menu__link">Contacts</a>
+            </li>
+          </ul>
+
+          <a
+            href="tel:+1 234 5555-55-55"
+            class="menu__call menu__call--phone"
+          >
+            +1 234 5555-55-55
+          </a>
+
+          <a
+            href="tel:+1 234 5555-55-55"
+            class="menu__call menu__call--action"
+          >
+            Book a test ride
+          </a>
+        </div>
+      </div>
+    </nav>
+
+    <main>
+      <section class="page__section about" id="about-us">
+        <div class="container">
+          <div class="grid grid--tablet">
+            <div class="grid__item grid__item--tablet-1-3 grid__item--desktop-1-4">
+              <h2 class="about__title">Move Free</h2>
+            </div>
+
+            <div class="grid__item grid__item--tablet-4-6 grid__item--desktop-7-12">
+              <p class="about__paragraph">
+                Shift your ride, not gears.
+                Find the fastest way to move in the city
+                as the bike adapts intuitively to power
+                the speed you need.
+              </p>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="page__section compare-bikes" id="compare-bikes">
+        <div class="container">
+          <h2 class="page__section-title">Compare bikes</h2>
+
+          <div class="grid grid--desktop">
+            <div class="grid__item grid__item--desktop-1-4">
+              <div class="bike">
+                <img
+                  class="bike__photo"
+                  src="./images/bikes/bike-1.svg"
+                  alt="Sporty_4"
+                >
+
+                <h3 class="page__section-subtitle bike__name">Sporty 4</h3>
+
+                <div class="bike__description">
+                  The iconic frame brought to a new performance
+                  height as a sporty, active ride.
+                </div>
+
+                <div class="page__section-subtitle bike__price">
+                  $ 2 590
+                </div>
+              </div>
+            </div>
+
+            <div class="grid__item grid__item--desktop-5-8">
+              <div class="bike">
+                <img
+                  class="bike__photo"
+                  src="./images/bikes/bike-2.svg"
+                  alt="Sporty_4"
```

Comment:
Add alt accordingly to the image
in all cses
```suggestion
                  alt="bycicle Ride in town ST"
```